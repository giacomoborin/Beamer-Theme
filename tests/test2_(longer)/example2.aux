\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\bibstyle{biblatex}
\bibdata{example2-blx,biblio}
\citation{biblatex-control}
\abx@aux@refcontext{nty/global//global/global}
\@writefile{toc}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lof}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lot}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\babel@aux{italian}{}
\babel@aux{italian}{}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {1}{1}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {0}{0}{2}{2/2}{}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {2}{2}}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\beamer@sectionintoc {1}{Titolo sezione numero 1}{3}{0}{1}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@sectionpages {1}{2}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@subsectionpages {1}{2}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\sectionentry {1}{Titolo sezione numero 1}{3}{Titolo sezione numero 1}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {1}{0}{1}{3/3}{}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {3}{3}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {1}{0}{2}{4/5}{}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {4}{5}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {1}{0}{3}{6/6}{}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {6}{6}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {1}{0}{4}{7/9}{}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {7}{9}}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\beamer@sectionintoc {2}{Altro titolo sezione}{10}{0}{2}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@sectionpages {3}{9}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@subsectionpages {3}{9}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\sectionentry {2}{Altro titolo sezione}{10}{Altro titolo sezione}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {2}{0}{1}{10/10}{}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {10}{10}}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\beamer@subsectionintoc {2}{1}{Sottosezione}{11}{0}{2}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@subsectionpages {10}{10}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{2}{1}{11}{Sottosezione}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {2}{1}{1}{11/12}{Sottosezione}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {11}{12}}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\beamer@subsectionintoc {2}{2}{Seconda sottosezione}{13}{0}{2}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@subsectionpages {11}{12}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@subsectionentry {0}{2}{2}{13}{Seconda sottosezione}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {2}{2}{1}{13/13}{Seconda sottosezione}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {13}{13}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {2}{2}{2}{14/14}{Seconda sottosezione}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {14}{14}}}
\@writefile{tdo}{\defcounter {refsection}{0}\relax }\@writefile{tdo}{\contentsline {todo}{Figure: immagine a caso}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {2}{2}{3}{15/15}{Seconda sottosezione}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {15}{15}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {2}{2}{4}{16/16}{Seconda sottosezione}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {16}{16}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {2}{2}{5}{17/17}{Seconda sottosezione}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {17}{17}}}
\citation{CF}
\abx@aux@cite{CF}
\abx@aux@segm{0}{0}{CF}
\citation{RAM}
\abx@aux@cite{RAM}
\abx@aux@segm{0}{0}{RAM}
\citation{SIN}
\abx@aux@cite{SIN}
\abx@aux@segm{0}{0}{SIN}
\citation{CF}
\abx@aux@segm{0}{0}{CF}
\citation{RAM}
\abx@aux@segm{0}{0}{RAM}
\citation{SIN}
\abx@aux@segm{0}{0}{SIN}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {2}{2}{6}{18/19}{Seconda sottosezione}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {18}{19}}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\beamer@sectionintoc {3}{Sezione finale}{20}{0}{3}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@sectionpages {10}{19}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@subsectionpages {13}{19}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\sectionentry {3}{Sezione finale}{20}{Sezione finale}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {3}{0}{1}{20/20}{}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {20}{20}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {3}{0}{2}{21/21}{}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {21}{21}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\slideentry {3}{0}{3}{22/22}{}{0}}}
\@writefile{nav}{\defcounter {refsection}{0}\relax }\@writefile{nav}{\headcommand {\beamer@framepages {22}{22}}}
\@writefile{nav}{\headcommand {\beamer@partpages {1}{22}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {20}{22}}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {20}{22}}}
\@writefile{nav}{\headcommand {\beamer@documentpages {22}}}
\@writefile{nav}{\headcommand {\gdef \inserttotalframenumber {17}}}
\abx@aux@refcontextdefaultsdone
\abx@aux@defaultrefcontext{0}{RAM}{nty/global//global/global}
\abx@aux@defaultrefcontext{0}{SIN}{nty/global//global/global}
\abx@aux@defaultrefcontext{0}{CF}{nty/global//global/global}
