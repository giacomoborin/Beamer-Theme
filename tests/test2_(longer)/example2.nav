\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {1}{1}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {0}{0}{2}{2/2}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {2}{2}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@sectionpages {1}{2}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionpages {1}{2}}
\defcounter {refsection}{0}\relax 
\headcommand {\sectionentry {1}{Titolo sezione numero 1}{3}{Titolo sezione numero 1}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {1}{0}{1}{3/3}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {3}{3}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {1}{0}{2}{4/5}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {4}{5}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {1}{0}{3}{6/6}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {6}{6}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {1}{0}{4}{7/9}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {7}{9}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@sectionpages {3}{9}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionpages {3}{9}}
\defcounter {refsection}{0}\relax 
\headcommand {\sectionentry {2}{Altro titolo sezione}{10}{Altro titolo sezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{0}{1}{10/10}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {10}{10}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionpages {10}{10}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionentry {0}{2}{1}{11}{Sottosezione}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{1}{1}{11/12}{Sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {11}{12}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionpages {11}{12}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionentry {0}{2}{2}{13}{Seconda sottosezione}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{1}{13/13}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {13}{13}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{2}{14/14}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {14}{14}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{3}{15/15}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {15}{15}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{4}{16/16}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {16}{16}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{5}{17/17}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {17}{17}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {2}{2}{6}{18/19}{Seconda sottosezione}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {18}{19}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@sectionpages {10}{19}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@subsectionpages {13}{19}}
\defcounter {refsection}{0}\relax 
\headcommand {\sectionentry {3}{Sezione finale}{20}{Sezione finale}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {3}{0}{1}{20/20}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {20}{20}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {3}{0}{2}{21/21}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {21}{21}}
\defcounter {refsection}{0}\relax 
\headcommand {\slideentry {3}{0}{3}{22/22}{}{0}}
\defcounter {refsection}{0}\relax 
\headcommand {\beamer@framepages {22}{22}}
\headcommand {\beamer@partpages {1}{22}}
\headcommand {\beamer@subsectionpages {20}{22}}
\headcommand {\beamer@sectionpages {20}{22}}
\headcommand {\beamer@documentpages {22}}
\headcommand {\gdef \inserttotalframenumber {17}}
