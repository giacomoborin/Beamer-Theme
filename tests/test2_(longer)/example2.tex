\documentclass{beamer}

\usepackage{lipsum}
\usepackage{todonotes}

%Macro for solving path problem in theme calling
\makeatletter
  \def\beamer@calltheme#1#2#3{%
    \def\beamer@themelist{#2}
    \@for\beamer@themename:=\beamer@themelist\do
    {\usepackage[{#1}]{\beamer@themelocation/#3\beamer@themename}}}

  \def\usefolder#1{
    \def\beamer@themelocation{#1}
  }
  \def\beamer@themelocation{}
  
\graphicspath{{../../}}

%todo: ovviamente, questo andrà cambiato
%\usepackage{../../beamerthemebeunitn}

\usefolder{../../}
\usetheme{beunitn}

\usepackage[italian]{babel}
\usepackage[utf8]{inputenc}

%Bibliography
\usepackage[style=numeric, maxnames=4,backend=bibtex]{biblatex}
% other styles: numeric authortitle
\addbibresource{biblio.bib}

\title{Un breve test}
\subtitle{esempio di applicazione del template, adesso allungo il titolo per vedere come viene}
\author{autori vari}
\institute{Università di Trento}
\date{30 febbraio 2042}

\begin{document}
% c'è un overfull hbox, ma tanto è solo un dummy test

\begin{frame}
  \titlepage
\end{frame}

\begin{frame}{Contenuti}
  \tableofcontents
\end{frame}

\section{Titolo sezione numero 1}

\begin{frame}
	Proviamo l'\alert{alert}
	\pause
	\begin{alertblock}{Titolo}
	Che blocco rosso
	\end{alertblock}
\end{frame}


\begin{frame}{Example Block}
	Ok proviamo un blocco:
	\begin{exampleblock}{Una bella formula}
		Questa è una bellissima formula:
		\[ e^{\pi i} + 1 = 0\]
	\end{exampleblock}
\end{frame}

\begin{frame}{Questo è un frame}
    \begin{block}{Un blocco di testo importante}
        \lipsum[66]
    \end{block}
    \pause
    \begin{itemize}
      \item \textbf{un punto}: posso scrivere cose
      \pause
      \item \textbf{altro punto}: posso scriverle due volte
    \end{itemize}
\end{frame}

\section{Altro titolo sezione}
\subsection{Sottosezione}
\begin{frame}{Questo è un frame dal titolo decisamente lungo}
    \textbf{Dal vangelo secondo Lipsum:} \\
    \pause
    \textit{\lipsum[75]} \\ 
    (Lipsum, 75)
\end{frame}

\subsection{Seconda sottosezione}

\begin{frame}{Formule}
    \[
    \mathcal L_{\mathcal T}(\vec{\lambda})
    = \sum_{(\mathbf{x},\mathbf{s})\in \mathcal T}
       \log P(\mathbf{s}\mid\mathbf{x}) - \sum_{i=1}^m
       \frac{\lambda_i^2}{2\sigma^2}
    \]
\end{frame}

\begin{frame}{Formule 2}
    \begin{align*}
        S(\omega) 
        &= \frac{\alpha g^2}{\omega^5} e^{[ -0.74\bigl\{\frac{\omega U_\omega         19.5}{g}\bigr\}^{\!-4}\,]} \\
        &= \frac{\alpha g^2}{\omega^5} \exp\Bigl[ -0.74\Bigl\{\frac{\omega       U_\omega 19.5}{g}\Bigr\}^{\!-4}\,\Bigr] 
    \end{align*}
\end{frame}


\begin{frame}{Il problema delle immagini}
    \missingfigure[figwidth=6cm]{immagine a caso}
\end{frame}

\begin{frame}{Conclusione}
    Iniziamo con un alertblock per attirare l'attenzione \dots
    \begin{alertblock}{Una frase profonda e filosofica}
        \lipsum[66]
    \end{alertblock}
    \beamerskipbutton{Bottone finto}
\end{frame}
\begin{frame}{}
    \dots
    ed ora un exampleblock per finire in bellezza:
    \begin{exampleblock}{Problemi di sonno?}
        "Chi ben mangia, dorme la sera." 
        \flushright
        (Orazio, forse)
    \end{exampleblock}
\end{frame}

\begin{frame}
	\nocite{CF} Come ha detto il caro \cite{RAM} \newline
	\pause
	\fullcite{SIN}
\end{frame}

\section{Sezione finale}

\begin{frame}{Bibliografia}
	\printbibliography[heading=none]
\end{frame}

\begin{frame}{Table of contents}
	\tableofcontents
\end{frame}

\end{document}


