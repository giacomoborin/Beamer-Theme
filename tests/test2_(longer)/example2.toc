\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax 
\babel@toc {italian}{}
\babel@toc {italian}{}
\defcounter {refsection}{0}\relax 
\beamer@sectionintoc {1}{Titolo sezione numero 1}{3}{0}{1}
\defcounter {refsection}{0}\relax 
\beamer@sectionintoc {2}{Altro titolo sezione}{10}{0}{2}
\defcounter {refsection}{0}\relax 
\beamer@subsectionintoc {2}{1}{Sottosezione}{11}{0}{2}
\defcounter {refsection}{0}\relax 
\beamer@subsectionintoc {2}{2}{Seconda sottosezione}{13}{0}{2}
\defcounter {refsection}{0}\relax 
\beamer@sectionintoc {3}{Sezione finale}{20}{0}{3}
