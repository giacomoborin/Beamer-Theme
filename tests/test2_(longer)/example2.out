\BOOKMARK [2][]{Outline0.1}{Titolo sezione numero 1}{}% 1
\BOOKMARK [2][]{Outline0.2}{Altro titolo sezione}{}% 2
\BOOKMARK [3][]{Outline0.2.1.11}{Sottosezione}{Outline0.2}% 3
\BOOKMARK [3][]{Outline0.2.2.13}{Seconda sottosezione}{Outline0.2}% 4
\BOOKMARK [2][]{Outline0.3}{Sezione finale}{}% 5
